appveyor.yml

```diff
@@ -22,4 +22,14 @@ build_script:
   - npm run test:e2e
   - npm run build
 
+on_success:
+  - ps: |
+      if ($env:APPVEYOR_REPO_COMMIT_MESSAGE -Match "pubchoco") {
```

Comment:
> Ugh, github lost my comment, so writing again...

Appveyor runs on every PR. In theory someone could PR a downgrade attack with `nuspec` pointing at older version.  What would Cocolatey do in that case? Reject? Override latest version with old one?

Reply:
Changed. However, couldn't someone just change the code? And mame a PR and hijack the releases anyways? There's a way to define scripts that does not include `.appveyor.yml` (see [docs](https://www.appveyor.com/docs/build-configuration/)). However, someone with enough knowledge could trigger the update too...

Reply:
Actually not, if we define like they do on the docs (on the UI itself), it won't be on `.appveyor.yml` thus it will be locked to `master`.